abc.templates/infra/contents/main.tf

```diff
@@ -31,6 +31,12 @@ module "REPLACE_MODULE_NAME" {
     editors = []
     viewers = []
   }
+  code_audit_dashboard = {
+    enabled = false
+    table_iam = {
+      viewers = []
+    }
```

Comment:
nit: why is this different from github_metrics_dashboard inputs? Should we try to unify on a common set of inputs here for dashboards? The might require some further discussion...

Reply:
I'm in agreement, they should have similar configurations. But that might require more refactoring, we can discuss this offline